velodrome/config.py

```diff
@@ -60,12 +60,33 @@ def main():
                 patch_configuration("grafana-config",
                                     project['grafana'],
                                     { "PROJECT": project_name })
-            for repository in project['repositories']:
-                print_deployments(["fetcher", "transform"], {
+            for repository, transforms in project['repositories'].items():
+                print_deployments(["fetcher"], {
                     "GH_ORGANIZATION": repository.split("/")[0],
                     "GH_REPOSITORY": repository.split("/")[1],
                     "PROJECT": project_name,
                 })
+                for metric, transformer in (transforms or {}).items():
+                    plugin = metric
+                    if "plugin" in transformer:
+                        plugin = transformer["plugin"]
+                    args = []
+                    if "args" in transformer:
+                        args = transformer["args"]
+                    apply_transform(args, {
+                        "GH_ORGANIZATION": repository.split("/")[0],
```

Comment:
There is probably a ton that can be done here, yeah. But this series is probably long enough already :D